src/auto-clicker.ts

```diff
@@ -22,7 +22,7 @@
 
   function clickBonus() {
     const interactiveElement = targetNode.querySelector<HTMLButtonElement>(
-      '.tw-interactive'
+      '.tw-button'
```

Comment:
```suggestion
      '.tw-button.tw-button--success'
```

this should avoid any conflicts with any buttons having `.tw-button` class

Reply:
I think this is superfluous. If the twitch changes anything, you will still need to change the class names.